cabal.project

```diff
@@ -211,8 +211,8 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger-specs
-  tag: 309178369eebe9ce02171987f6d23c7778f7db26
-  --sha256: 1a9cm3ykz9dzfrfv4yk8a2rw5pj4frlwrd6rpp97j5n8rrx34v5k
+  tag: 4d6907f3d820aa52ab0521cf981fa63c7876adc6
```

Comment:
yep, this commit is great

Reply:
I just changed it to `10a6f5fe03c8d48141a750747dfe6b1cd77b41cd` -- also OK, right?

Reply:
yes, `10a6f5fe03c8d48141a750747dfe6b1cd77b41cd` is great as well.

Reply:
`12a0ef69d64a55e737fbf4e846bd8ed9fb30a956` is even greater (I merged w/o bors in CLS)

Reply:
Bah. OK. now it's `12a0ef69d64a55e737fbf4e846bd8ed9fb30a956`.

But that's the same as `10a6f5fe03c8d48141a750747dfe6b1cd77b41cd` (the sha didn't even change).